proto/one/common/model/model_input.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+import "model_state.proto";
+import "model_parameter.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message ModelInput {
```

Comment:
I think this could be named better, it's not really a model input, the model input is part of this model, as is the model itself or model identifier.  I'm not sure I have much as far as suggestions though, perhaps ModelExecution or ModelRun as I think the intent is to send this object to the endpoint that will run/execute the model.  The model itself is a process model so maybe that could be incorporated into the name?

Reply:
I guarantee that there will be another round or two of changes needed for this model.  I think getting the data science team something to work with on the sooner side will help answer a lot of questions.